CHANGELOG.md

```diff
@@ -9,19 +9,43 @@
 
 # UNRELEASED
 
+# UNRELEASED v.1.32.0
+
+See https://github.com/filecoin-project/lotus/blob/release/v1.32.0/CHANGELOG.md
+
+# Node v1.31.1-rc1 / 2025-01-20
+
+This Lotus release candidate introduces several new features and improvements, including JSON output for tipsets in `lotus chain list` cmd, enhanced logging during network upgrade migrations, and additional Bootstrap nodes. It also includes a refactored Ethereum API implementation into smaller, more manageable modules in a new `github.com/filecoin-project/lotus/node/impl/eth` package, as well as adding network name as a tag in most metrics - making it easier to create Graphana Dashboards for multiple networks. Please review the upgrade warnings and documentation for any important changes affecting RPC providers, node operators, and storage providers.
+
+## ☢️ Upgrade Warnings ☢️
+- If you are running the v1.30.x version of Lotus, please go through the Upgrade Warnings section for the [v1.30.* releases](https://github.com/filecoin-project/lotus/releases/tag/v1.31.0) and [v1.31.*](https://github.com/filecoin-project/lotus/releases/tag/v1.31.0) before upgrading to this release.
```

Comment:
Why `[v1.30.* releases](https://github.com/filecoin-project/lotus/releases/tag/v1.31.0)`

v1.30.* is referencing https://github.com/filecoin-project/lotus/releases/tag/v1.31.0
1.30.* with 1.31.0

Reply:
That is an copy+paste error yes - fixed in: https://github.com/filecoin-project/lotus/pull/12833/commits/e09e988a8ccda7a50e7ac16a1455cb3b7fb26b96